packages/purpose-process/test/integration/getPurposeById.test.ts

```diff
@@ -72,11 +72,11 @@ describe("getPurposeById", () => {
     await addOneEService(mockEService);
     await addOneTenant(producer);
 
-    const result = await purposeService.getPurposeById(
+    const { data } = await purposeService.getPurposeById(
```

Comment:
Same for these tests in this file, let's test the entire returned object including metadata